package-lock.json

```diff
@@ -11,6 +11,9 @@
       "workspaces": [
         "packages/*"
       ],
+      "dependencies": {
```

Comment:
You can safely revert this file back, the package lock in the cdn folder is the only one that needs updating 👍